docs/developer_guide/OpenSCENARIOSupport.md

```diff
@@ -376,7 +376,7 @@ See also section [Scoping](#scoping).
 | RelativeWorldPosition  | Unsupported |
 | RelativeObjectPosition | Unsupported |
 | RoadPosition           | Unsupported |
-| RelativeRoadPosition   | Unsupported |
+| RelativeRoadPosition   |      ✔      |
```

Comment:
What this pull request introduces is RelativeObjectPosition, not RelativeRoadPosition.

Reply:
oh, sorry for the mistake, fixed in https://github.com/tier4/scenario_simulator_v2/pull/1030/commits/7ef4ad3287dd981d61006b2d6f5ff0a90d0d0f64